### ISSUE TITLE
the loss is NAN when train model

### ISSUE THREAD

**Issue by WHQCHINA** [preddy5/Im2Vec#10]
![image](https://user-images.githubusercontent.com/10349500/127615023-d4b0aa26-2d26-44fe-b318-fa9ede1d0500.png)

**Comment by preddy5** [comment]
Hey @WHQCHINA 
This tends to happen when you have a high learning rate, could try training with a lower running rate and check if that resolves the issue.

**Comment by WHQCHINA** [comment]
> Hey @WHQCHINA
> This tends to happen when you have a high learning rate, could try training with a lower running rate and check if that resolves the issue.

@preddy5 The result is NAN use your way. find function training_step  run failed

**Comment by WHQCHINA** [comment]
@preddy5 what is the true label  about train data. can you give me to your train data?

**Comment by preddy5** [comment]
Hey @WHQCHINA 
were you able to replicate the results with the provided emoji dataset?

**Comment by WHQCHINA** [comment]
> Hey @WHQCHINA
> were you able to replicate the results with the provided emoji dataset?

![image](https://user-images.githubusercontent.com/10349500/127939048-160af668-16b2-43ed-9fb4-aaf45886ff51.png)
The train loss still is NAN use dataset in train. And print log in training_step function, that is print function not run.

**Comment by preddy5** [comment]
@WHQCHINA which version of pytorch-lightning are you using?

**Comment by WHQCHINA** [comment]
> @WHQCHINA which version of pytorch-lightning are you using?

version of pytorch-lightning update to 0.9.0, it's work, thanks!
